python/tvm/relay/op/op_attrs.py

```diff
@@ -21,24 +21,233 @@
 
 @register_relay_attr_node
 class Conv2DAttrs(Attrs):
-    """Attribute of nn.conv2d"""
+    """Attributes for nn.conv2d"""
 
 
 @register_relay_attr_node
 class Conv2DWinogradAttrs(Attrs):
-    """Attribute of nn.contrib_conv2d_winograd_without_weight_transform"""
+    """Attributes for nn.contrib_conv2d_winograd_without_weight_transform"""
 
 
 @register_relay_attr_node
 class Conv2DWinogradWeightTransformAttrs(Attrs):
-    """Attribute of nn.contrib_conv2d_winograd_weight_transform"""
+    """Attributes for nn.contrib_conv2d_winograd_weight_transform"""
 
 
 @register_relay_attr_node
 class Conv2DWinogradNNPACKWeightTransformAttrs(Attrs):
-    """Attribute of nn.contrib_conv2d_winograd_nnpack_weight_transform"""
+    """Attributes for nn.contrib_conv2d_winograd_nnpack_weight_transform"""
 
 
 @register_relay_attr_node
 class GlobalPool2DAttrs(Attrs):
-    """Attribute of nn.global_pool"""
+    """Attributes for nn.global_pool"""
+
+
+@register_relay_attr_node
```

Comment:
I needed them for this one because of the way op calls are handled: If the attr node is not registered, then it is treated as just a tvm.NodeBase object and there is thus no `keys()` method or any way to iterate through the attribute values.

Reply:
I no longer depend on these changes so I will move them to another PR as requested.